package.json

```
@@ -3,21 +3,17 @@
   "version": "1.0.0",
   "description": "packages for kintone customizations",
   "private": true,
-  "workspaces": [
-    "packages/*",
-    "examples/*"
-  ],
   "scripts": {
-    "prebuild": "lerna run prebuild --parallel",
-    "build": "tsc --build packages --force",
-    "postbuild": "lerna run postbuild --parallel",
-    "lint": "lerna run lint --parallel",
-    "fix": "lerna run fix --parallel",
+    "prebuild": "pnpm -r --parallel run prebuild",
+    "build": "pnpm prebuild && tsc --build packages --force && pnpm postbuild",
+    "postbuild": "pnpm -r --parallel run postbuild",
+    "lint": "pnpm -r --parallel run lint",
+    "fix": "pnpm -r --parallel run fix",
     "pretest": "jest",
-    "start": "yarn build --watch",
-    "test": "lerna run test --parallel",
-    "pretest:ci": "yarn pretest",
-    "test:ci": "lerna run test:ci --stream"
+    "start": "pnpm build --watch",
+    "test": "pnpm -r --parallel run test",
+    "pretest:ci": "pnpm pretest",
+    "test:ci": "pnpm pretest && pnpm -r --stream run test:ci"
```

It's better to rename `pretest` to `test:root`. (because it is  not the preparing step of test)
Also you can remove `pretest:ci`.

```suggestion
    "start": "pnpm build --watch",
    "test": "pnpm test:root && pnpm -r --parallel run test",
    "test:ci": "pnpm pretest && pnpm -r --stream run test:ci"
```

---

@mshrtsr 
Thanks for suggesting detail code. I have a question:
Do we need `pnpm test:root` in this:
"test": "pnpm test:root && pnpm -r --parallel run test",

Because "test" script in each package.json has already done it.
Thanks for answering.

---

@tuanphamcybozu 
Thank you for asking!
The `test:root` script is to run tests in `__tests__` of the project root directory.

---

@mshrtsr 
ah I got it.
Thanks!